libraries/azure_service_bus_namespace.rb

```diff
@@ -0,0 +1,22 @@
+require 'azure_generic_resource'
+
+class AzureServiceBusNamespace < AzureGenericResource
+  name 'azure_service_bus_namespace'
+  desc 'Retrieves and verifies the settings of an Azure Service Bus Namespace.'
+  example <<-EXAMPLE
+    describe azure_service_bus_namespace(resource_group: 'migrated_vms', name: 'inspec_ns') do
```

Comment:
```suggestion
    describe azure_service_bus_namespace(resource_group: 'RESOURCE_GROUP',  name: 'SERVICE_BUS_NAMESPACE') do
```

Reply:
as discussed offline we don't follow this construct in ruby files